src/DataTable.vue

```diff
@@ -255,6 +253,36 @@
 		},
 
 		computed: {
+			perPageOptions: function() {
```

Comment:
Thank you, I edited the readme for the "typo" and used the shorter way for options